app/src/main/java/com/github/wnder/GuessLocationActivity.java

```diff
@@ -128,6 +138,9 @@ public void onStyleLoaded(@NonNull Style style) {
      */
     @Override
     public boolean onMapClick(@NonNull LatLng point) {
+        if (guessConfirmed) {
+            return true;
```

Comment:
I know it's one line but wouln't this be tested if you just clicked once more in the tests?